prompt_toolkit/styles/defaults.py

```diff
@@ -193,7 +197,7 @@
     "pygments.literal.number": "#666666",
     "pygments.generic.heading": "bold #000080",
     "pygments.generic.subheading": "bold #800080",
-    "pygments.generic.deleted": "#a00000",
+    "pygments.generic.deleted": "strike",
```

Comment:
This is the only change where I'm not sure whether it's a good idea. These class names are used by Pygments for rendering diffs. 1. underline is probably not everywhere that well supported. 2. people are used to red/green. So, I think I'll revert this line and choose the safe default.

Reply:
I don't have any objections at all to change this back to `"a00000"`, if it is the better default!

Reply:
@jonathanslenders: I notice that the commit https://github.com/prompt-toolkit/python-prompt-toolkit/commit/81fbd23172108584d47f8403b5954ec91971096c still has this line with `"strike"` instead of `"#a00000"`, because I have never changed it back, since I had the feeling that our discussion has not resolved this issue. Please feel free to change it back to `"#a00000"` …